PocketKit/Sources/Sync/ApolloClientProtocol+Extensions.swift

```diff
@@ -16,30 +16,50 @@ public extension ApolloClientProtocol {
         )
     }
 
-    func fetch<Query: GraphQLQuery>(query: Query) async throws -> GraphQLResult<Query.Data> {
+    func fetch<Query: GraphQLQuery>(query: Query, filename: String = #file, line: Int = #line, column: Int = #column, funcName: String = #function) async throws -> GraphQLResult<Query.Data> {
```

Comment:
The filename stuff passes through the original functional caller to the log statement, its more useful to have the original network logger in the logs instead of the ApolloClient extensions file.